### PR TITLE
Generator questions and Archetype

### PR DESCRIPTION
**Pull request by scott-wyatt**
Initiate Archetype and update questions to mimic generator-trails

- [x] Include generator-trails defaults and wrap with generator-treefrog.
- [ ] Test: app needs to pass.
- [x] Merge all treefrog selection `package.json` and install with NPM.
- [x] Require generator-trails questions instead of rewriting them.

### CODE REVIEW

File: src/app/questions/index.js [hunk 1]
```diff
@@ -1,11 +1,84 @@
 'use strict';
 
-Object.defineProperty(exports, "__esModule", {
-  value: true
-});
-
 //Questions
-exports.default = [
+export default [
+
```
Comment (jaumard) [inline src/app/questions/index.js]: Maybe there a way to `require` questions from trails-generator instead of hard code them here again ?

Comment (scott-wyatt) [inline src/app/questions/index.js]: Yes, I think requiring them is much better idea.

**Comment by jaumard** [comment]
Ok it's a very big PR :D I think it's all good let's merge, I have to test it a little to see behavior ect... but sound pretty cool :)

**Comment by scott-wyatt** [comment]
Yeah, I'm still having issues with it passing tests.  But it's getting there. Enough to start more work on.

**Pull request merged**